batched/sparse/src/KokkosBatched_CrsMatrix.hpp

```diff
@@ -53,6 +53,7 @@ class CrsMatrix {
   KOKKOS_INLINE_FUNCTION
   ~CrsMatrix() {}
 
+#ifndef DOXY
```

Comment:
Is this hiding the comment block when Doxygen is running?

Reply:
Yes. With my latest force push, this is no longer needed.